### PR TITLE
This patch needs to be rolled back

### PR DESCRIPTION
**Pull request by scstraus**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
I'm not sure what I am looking at? 
This PR add something, but isn't rolling back anything?

Could you update the PR description to explain what is happening and the intend of this PR? Thanks! 👍

**Comment by scstraus** [comment]
Sorry I stink at Github. To the point where I think I probably should stop contributing, because I'm just too bad at it. The point is that the scstraus:patch-3 were edits made to the wrong page, and so all of the changes that were made in that patch need to be removed from the page. If this is the wrong way of proposing that, I can go in and try editing the current version to remove the parts that were added, I was just afraid that I'd do that wrong, and this way at least you 100% know exactly what changes need to be rolled back.. 

Basically I'm just asking you to undo [20395](https://github.com/home-assistant/home-assistant.io/pull/20395). I've opened a separate PR [20810](https://github.com/home-assistant/home-assistant.io/pull/20810) to put these changes into the correct page.
